setup/l10n_it_fatturapa/setup.py

```diff
@@ -2,5 +2,11 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
+    odoo_addon={
+        'external_dependencies_override': {
+            'python': {
+                'pyxb': 'PyXB==1.2.6',
```

Comment:
```suggestion
                'pyxb': 'PyXB>=1.2.6',
```

Unless there is a very good reason,, never pin versions here, because it could create conflict with other modules requiring a different version of the same library.

Reply:
I'll wait other reviers to decide how to set it @eLBati @sergiocorato

Reply:
Thanks @sbidoul for the review!
Here I think we have a good reason because we are using PyXB to generate Python files from an `XSD` schema, and the generated files explicitly ask for the PyXB version that has been used to generate them:https://github.com/OCA/l10n-italy/blob/c5dfaffb8e64713a02d2ef29a50de47709e7a950/l10n_it_fatturapa/bindings/_ds.py#L31-L34